release/README.md

```diff
@@ -58,21 +63,71 @@ tweak as needed.
 - A change that affects all libraries should only be documented in the
   `Common Libraries` section.
 
-### Run checkers
+#### Run checkers
 
 ```bash
 ci/cloudbuild/build.sh -t checkers-pr
 ```
 
-### Send a PR with all these changes
+#### Send a PR with all these changes
+
+```bash
+git add .
+git checkout -b release-changelog
+git commit -am "docs(release): update changelog for XX-20XX release" // Change this to the month of the release
```

Comment:
s/XX-20XX release/$(date +%Y-%m) release/ and you can also `s;//.*;;`.  In any case `# ` is the comment character for bash.

Reply:
Done.

I don't know what you mean by `s;//.*;;.`.

Reply:
That you could remove the comment... which you did.